sccm/apps/deploy-use/create-deploy-scripts.md

```diff
@@ -107,6 +110,9 @@ After a script is approved, it can be run against a collection you choose.
 >[!IMPORTANT]
 >The script is given a one-hour time period in which to run. If it does not run (for example if the PC is turned off) in this time period, you must run it again.
 
+>[!IMPORTANT]
+>The script is executed as the system or computer account on the targeted client(s). This account has very limited network access. Any access to remote systems and\or locations by the script must be provisioned with this in mind.
```

Comment:
> [!IMPORTANT] 
> The script is executed as the system or computer account on the targeted client(s). This account has very limited network access. Any access to remote systems and locations by the script must be provisioned with this in mind.

Reply:
Thanks ... "and\or" are difficult for localization.

Reply:
Thanks @mattbriggs